src/plugin/settings/engine.cpp

```diff
@@ -527,14 +525,14 @@ void Engine::doGetSettingsOnActiveSettingChange(Result<const IActiveSettingChang
 
     std::map<std::string, std::string> values = toStdMap(shareToPtr(activeSettingChangedAction->settingsValues()));
 
-    if (!processActiveSettings(&model, &values, {settingId}))
-    {
-        std::string errorMessage = "Unable to process the active settings section";
-        NX_PRINT << errorMessage;
-        *outResult = error(ErrorCode::internalError, errorMessage);
+    // if (!updateModel(&model, &values, {settingId}))
```

Comment:
Do you still want to leave this commented out sections?

Reply:
No, I'll drop it.